frontend/src/scenes/feature-flags/activityDescriptions.tsx

```diff
@@ -260,6 +260,22 @@ export function flagActivityDescriber(logItem: ActivityLogItem, asNotification?:
         return { description: null }
     }
 
+    if (logItem.activity === 'created') {
```

Comment:
drive by PR when I noticed that we never actually visualize (but we _do_ log) creation events in the feature flag history!  Here's what it look like now

<img width="815" alt="image" src="https://github.com/user-attachments/assets/06b02d7c-aa6c-4a1c-83fc-be6520be4fb0">